views/content/wallet.md

```diff
@@ -0,0 +1,14 @@
+## Ethereum Wallet
+
+The Ethereum Wallet app that you can download allows you to run the full block chain on your own computer. It allows you very easy to be part of the blockchain.
+
+![](/images/wallet-screenshot.png)
+
+With the wallet app you can create new wallets and transfer ether really easily. You can also interact with smart contracts that other have deployed to the Ethereum blockchain.
```

Comment:
Ethereum Wallet allows you to receive and transfer ether or any ethereum compatible token, as well as allowing you to deploy a contract or interact with any smart contract deployed on the network. The Wallet supports the Main ethereum chain, test chains and even private networks.